.github/workflows/dependabotautomerge.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: auto-merge
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5 # v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@5d98f10d910782d0bb431fc03ae26532543ae184 # v2
```

Comment:
can we have this specific action a special case where dependabot PRs are not automatically merged? Since it has high privileges, we should be give en the change to review those, otherwise this action could merge anything they want

Reply:
actually I'm confused what this action does, because the description of the PR says `Dependabot will wait until all your status checks pass before merging. This is a function of Dependabot itself, and not this Action.`